modules/axhal/src/lib.rs

```diff
@@ -77,3 +77,6 @@ pub use self::platform::platform_init;
 
 #[cfg(feature = "smp")]
 pub use self::platform::platform_init_secondary;
+
+#[cfg(all(target_arch = "x86_64", frature = "alloc"))]
```

Comment:
Create a new `mod pci` to place PCI-related APIs:

```Rust
pub mod pci {
    #[cfg(...)]
    pub use self::platform::acpi::{get_ecam_address, get_pci_irq_vector};
}
```

Reply:
OK. And i added related functions to `dummy` platform in order to pass doc.